ambry-api/src/main/java/com/github/ambry/config/FrontendConfig.java

```diff
@@ -201,6 +203,10 @@
   @Default("20 * 1024 * 1024")
   public final int maxJsonRequestSizeBytes;
 
+  @Config(ENABLE_UNDELETE)
```

Comment:
Why create this frontend config `ENABLE_UNDELETE`? Looks like you already have a  `ROUTER_ENABLE_UNDELETE` in the other PR?

Reply:
I've removed the configuration in the router. This is the gateway for undelete, to control it here is better.